libseccomp/src/error.rs

```diff
@@ -16,41 +32,59 @@ pub enum ErrorKind {
     Source,
 }
 
-#[derive(Debug)]
 pub struct SeccompError {
     kind: ErrorKind,
-    code: Option<i32>,
     source: Option<Box<dyn Error + Send + Sync>>,
 }
 
-impl fmt::Display for SeccompError {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let msg = match &self.kind {
+impl SeccompError {
+    fn msg(&self) -> String {
```

Comment:
```suggestion
    fn msg(&'a self) -> &'a str {
```
Does this work?

If not, we could consider
```suggestion
    fn msg(&self) -> Cow<'static, str> {
```

or we just say that the code is cold enough that it is not worth to optimize it.

Reply:
I use `Cow`. Thanks.